### DISCUSSION TITLE
[Feature] Discord rich embeds

### DISCUSSION THREAD

**Discussion by nickXIII** [immich-app/immich#3056] [Feature Request]
### The feature

I'd love the ability to embed uploads in Discord, especially GIF and webm files. Imgur has this ability simply by pasting the direct image link. See the example uploaded below.
[webmtest.webm](https://github.com/immich-app/immich/assets/33738783/840a478d-f74e-4ea1-82a5-e5c87ef6ae8f)

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Embeds on shared links are already supported.

**Reply by nickXIII** [reply]
Embeds, yes, but rich embeds where the content plays or can be expanded without visiting the link? Unless I'm doing something wrong on my end, that doesn't seem to be the case. I created a share link for both a .gif, and a .webm animation and tested. Neither animate in the embed, the link has to be clicked to view it.

![image](https://github.com/immich-app/immich/assets/33738783/4398e1e3-e2ac-4fa4-9867-e9fce4729f31)

**Comment by zdimension** [comment]
This request was erroneously marked as closed. Immich doesn't produce rich embeds, so videos can't be played directly in Discord. This would really be nice to have
